color-palettes.json

```
@@ -197,257 +193,257 @@
   },
   "Bitcoin": {
     "hex": "#F7931A",
-    "aliases": ["bitcoin"]
+    "aliases": ["bitcoin", "btc"]
   },
   "Fantom": {
     "hex": "#1969FF",
-    "aliases": ["fantom"]
+    "aliases": ["fantom", "ftm"]
   },
   "Avalanche": {
     "hex": "#E84142",
-    "aliases": ["avalanche_c"]
+    "aliases": ["avalanche_c", "avalanche", "avalanche c-chain", "avax"]
   },
   "Arbitrum": {
     "hex": "#12AAFF",
-    "aliases": ["arbitrum"]
+    "aliases": ["arbitrum", "arb"]
   },
   "Solana": {
     "hex": "#14F195",
     "aliases": ["solana", "sol"]
   },
-  "bnb": {
+  "BNB": {
     "hex": "#F3BA2F",
-    "aliases": ["bnb"]
+    "aliases": ["bnb", "bsc"]
   },
   "Optimism": {
     "hex": "#FF0420",
-    "aliases": ["optimism"]
+    "aliases": ["optimism", "op"]
   },
-  "mint": {
+  "Mint": {
     "hex": "#31bf54",
     "aliases": ["mint"]
   },
-  "zero_network": {
+  "Zero Network": {
     "hex": "#FE5161",
     "aliases": ["zero_network"]
   },
   "xai": {
     "hex": "#FD0132",
     "aliases": ["xai"]
   },
-  "wemix": {
+  "Wemix": {
     "hex": "#5759F7",
     "aliases": ["wemix"]
   },
-  "swellchain": {
+  "Swellchain": {
     "hex": "#2f43ec",
-    "aliases": ["swellchain"]
+    "aliases": ["swellchain", "swell"]
   },
-  "superposition": {
+  "Superposition": {
     "hex": "#2ff5d6",
     "aliases": ["superposition"]
   },
-  "redstone": {
+  "Redstone": {
     "hex": "#AE0822",
     "aliases": ["redstone"]
   },
-  "rari": {
+  "Rari": {
     "hex": "#963BFF",
     "aliases": ["rari"]
   },
-  "proof_of_play": {
+  "Proof of Play": {
     "hex": "#6c41ff",
     "aliases": ["proof_of_play"]
   },
-  "proof_of_play_boss": {
+  "Proof of Play Boss": {
     "hex": "#6c41ff",
     "aliases": ["proof_of_play_boss"]
   },
-  "omni": {
+  "Omni": {
     "hex": "#1767f3",
     "aliases": ["omni"]
   },
-  "metis": {
+  "Metis": {
     "hex": "#04d1ff",
     "aliases": ["metis"]
   },
-  "monad_testnet": {
+  "Monad Testnet": {
     "hex": "#836EF9",
     "aliases": ["monad_testnet"]
   },
-  "lisk": {
+  "Lisk": {
     "hex": "#4a36b4",
     "aliases": ["lisk"]
   },
-  "hychain": {
+  "Hychain": {
     "hex": "#fa86ca",
     "aliases": ["hychain"]
   },
-  "ham": {
+  "Ham": {
     "hex": "#eb4747",
     "aliases": ["ham"]
   },
-  "funkichain": {
+  "Funkichain": {
     "hex": "#b2eb15",
     "aliases": ["funkichain"]
   },
-  "cyber": {
+  "Cyber": {
     "hex": "#0bdc0e",
     "aliases": ["cyber"]
   },
-  "avalanche_fuji": {
+  "Avalanche Fuji": {
     "hex": "#ffc3c3",
     "aliases": ["avalanche_fuji"]
   },
-  "ancient8": {
+  "Ancient8": {
     "hex": "#D8FF76",
     "aliases": ["ancient8"]
   },
-  "assethub_kusama": {
+  "AssetHub Kusama": {
```

```suggestion
  "AssetHub_Kusama": {
```

I know this is not ideal, but let's go with it now if possible